tests/crossover.rs

```diff
@@ -0,0 +1,59 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use core::convert::TryInto;
+
+use dusk_jubjub::JubJubScalar;
+use dusk_pki::SecretSpendKey;
+use phoenix_core::{Error, Message, Note};
+
+#[test]
+fn crossover_hash() -> Result<(), Error> {
+    let rng = &mut rand::thread_rng();
+
+    let ssk = SecretSpendKey::random(rng);
```

Comment:
See my comment about `rand-core` / `rand` in `dev-dependencies`. I believe `dusk-pki` is using `rand-core` so we shouldn't have the need to use `rand` as deps, and we should be able to use `OsRng` directly.

Reply:
Fixed